guvnor-structure/guvnor-structure-backend/src/main/java/org/guvnor/structure/backend/repositories/git/GitMetadataStoreImpl.java

```diff
@@ -33,7 +33,7 @@
 
     private Logger logger = LoggerFactory.getLogger(GitMetadataStoreImpl.class);
     public static final String SEPARATOR = "/";
-    public static final String METADATA = "default://system/metadata";
+    public static final String METADATA = "default://metadata";
```

Comment:
Do we need to worry ourselves with the migration of data from the _old_ (location) metadata repository to the _new_?